examples/codecpreferences/src/index.js

```diff
@@ -185,18 +185,23 @@ function wait(ms) {
   // media should join.
   roomName = someRoom.name;
 
-  // Attach the newly subscribed Track to the DOM.
-  someRoom.on('trackSubscribed', attachTrack.bind(
-    null,
-    audioPreview,
-    videoPreview,
-    showAppliedCodec.bind(null, someRoom)));
-
-  // Detach the unsubscribed Track from the DOM.
-  someRoom.on('trackUnsubscribed', detachTrack.bind(
-    null,
-    audioPreview,
-    videoPreview));
+  // When a Participant joins the Room...
+  someRoom.on('participantConnected', function(participant) {
```

Comment:
@makarandp0 I don't think we need to change the examples. Here, we are focusing more on the specific use case, and not on Room setup.

Reply:
okay, I will revert changes to this example